### PR TITLE
updated /path/to/your/project to your_project_path in installation.md

### PR DESCRIPTION
**Pull request by Jkishan544**
Targets #2020 

### Checklist:

- [ ] I have performed a self-review of my own code.

### CODE REVIEW

File: docs/installation.md [hunk 1]
```diff
@@ -144,8 +144,8 @@ Before you start following the guidelines, make sure to go through the [prerequi
             ```apacheconf
             <VirtualHost *:80>
                 ServerName portal.test
-                DocumentRoot "/path/to/your/project"
-                <Directory "/path/to/your/project">
+                DocumentRoot "your_project_path"
```
Comment (P4NK4J) [inline docs/installation.md:147]: You don't need to change this, instead the sentence in line number 143, should be Copy the absolute file path for the `public` directory of the project and paste it below where `/path/to/your/project` is written

Please make these changes and then assign for review

**Review by P4NK4J — CHANGES_REQUESTED** [review]
Left a comment, please check

File: docs/installation.md (hunk 1 above)
Comment (Jkishan544) [reply docs/installation.md]: i have changed

**Review by P4NK4J — APPROVED** [review]
Looks good.

**Pull request merged**